MultitagInspector.cs

```diff
@@ -122,7 +123,11 @@ public override void OnInspectorGUI()
                 currentNewTag = "";
                 EditorUtility.SetDirty(tagObj);
             }
- 
+            else if (Multitag.AllTags.Count > 0)
+            {
+                tagObj.Tags.Add(Multitag.AllTags[0]);
```

Comment:
consider keeping track of the last tag a user selected, and defaulting to that one if it still exists